Source/WebGPU/WebGPU/CommandEncoder.swift

```diff
@@ -1854,15 +1854,26 @@ extension WebGPU.CommandEncoder {
             self.generateInvalidEncoderStateError()
             return
         }
-        guard self.validateResolveQuerySet(querySet: querySet, firstQuery: firstQuery, queryCount: queryCount, destination: destination, destinationOffset: destinationOffset) else {
+
+        guard self.validateResolveQuerySet
```

Comment:
Patch looks good, but looks like the Swift compiler is unhappy with the formatting here